core/src/main/java/org/apache/gravitino/storage/relational/mapper/provider/postgresql/SecurableObjectPostgreSQLProvider.java

```diff
@@ -160,7 +154,7 @@ public String deleteSecurableObjectsByLegacyTimeline(
     return "DELETE FROM "
         + SECURABLE_OBJECT_TABLE_NAME
         + " WHERE id IN (SELECT id FROM "
-        + ROLE_TABLE_NAME
+        + SECURABLE_OBJECT_TABLE_NAME
```

Comment:
@jerqi could you plz help to confirm this change?

Reply:
cc @yuqi1129

Reply:
Yes, it should be a bug caused by #6395 and the current fix is okay.